acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PassportPageObject.java

```diff
@@ -310,8 +310,10 @@ public void searchForUATUser(String number) {
     public void navigateToPassportResponse(String validOrInvalid) {
         assertURLContains("callback");
         if ("Invalid".equalsIgnoreCase(validOrInvalid)) {
+            BrowserUtils.waitForVisibility(errorResponse, 5);
```

Comment:
Nice one, does this mitigate the random test failures?

Reply:
Yes this was for the div appearing after click, have also updated the PR so it also waits for the result page to be loaded before doing the above checks.